dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/internal/BaseDimensionDataCloudControlMockTest.java

```diff
@@ -217,4 +225,15 @@ protected void assertBodyContains(RecordedRequest recordedRequest, String expect
          throw Throwables.propagate(e);
       }
    }
+
+   private List<NameValuePair> parsePath(String fullPath) {
+      return URLEncodedUtils.parse(fullPath, Charset.defaultCharset(), '?', '&');
+   }
+
+   private void assertParameters(List<NameValuePair> parsedPath, List<NameValuePair> parsedRequest) {
+      assertEquals(parsedPath.size(), parsedRequest.size());
+      for (NameValuePair parameter : parsedPath) {
+         assertTrue(parsedRequest.contains(parameter));
+      }
+   }
```

Comment:
I don't really get the value in comparing the requests like this. Can't just we compare the strings and avoid the need of adding an additional dependency just for this?
Also, this check does not guarantee paths are the same; it does not check the order of the value pairs. Consider just comparing the requests path strings, as we do in all other providers.